### PR TITLE
make konva run in node 0.12 atleast

### PR DESCRIPTION
**Pull request by hugomrdias**

### CODE REVIEW

File: src/Global.js [hunk 1]
```diff
@@ -29,7 +29,7 @@
  */
 
 // runtime check for already included Konva
-(function(){
+(function(global){
```
Comment (lavrton) [inline src/Global.js]: Why do we need `global`?

Comment (lavrton) [inline src/Global.js]: Global detection already included in https://github.com/hugomrdias/konva/blob/node/src/Global.js#L203

Comment (hugomrdias) [inline src/Global.js]: global is undefined inside IIFE

**Comment by kanzelm3** [comment]
+1

**Comment by kanzelm3** [comment]
This fixes an issue I am having as well, please merge this in.

**Pull request merged**